.travis.yml

```diff
@@ -2,7 +2,9 @@ language: python
 python:
 - '2.7'
```

Comment:
I thought about it, but then it would also run your coverage tests in 3.4, and I don't know if that would cause conflicts with your coveralls plugin...